docs/skopeo-list-tags.1.md

```diff
@@ -121,8 +124,21 @@ $ skopeo list-tags docker://localhost:5000/fedora
 
 ```
 
+### Docker-archive Transport
+
+```sh
+skopeo list-tags docker-archive:/root/busybox.tar.gz
```

Comment:
Suggestion: include prompt on all sample commands. Since there's no reason to make this root-only, I would encourage you to use `$` and to change the path as well (to `/tmp/` instead of `/root`).